src/modules/components/LanguageSwitcher.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box, Button } from "@mui/material";
+import { useTranslation } from "react-i18next";
+
+const LanguageSwitcher = () => {
+  const { i18n } = useTranslation();
+
+  const changeLanguage = (lng: string) => {
+    i18n.changeLanguage(lng);
+  };
+
+  return (
+    <Box>

```

Comment:
Please use a corresponding component to show which language is currently selected. Please add it to the store as well.